File: codes/java/temperature/Temperature.java
import java.util.*;

class Temperature {
    public static void main(String[] args) {
        float c, f;
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the temperature in celcius: ");
        c = sc.nextInt();
        f = (c*9/5) + 32;
        sc.close();
        System.out.println("The temperature in fahrenheit is " + f);
    }
}

File: codes/python/temperature/temperature.py
value = input("Enter the value of temperature: ")
temperature = input("Enter the unit in Celcius(c) or Fahrenheit(h): ")

if temperature == str('c'):
    answer = (int(value) * 9/5) + 32
    print("The value in Fahrenheit: " + str(answer) + "F")

elif temperature == str('h'):
    answer = (int(value) -32) * 5/9
    print("The value in Celcius: " + str(answer) + "C")

else:
    print("Invalid unit of temperature")